app/services/clients/adminusers.client.js

```diff
@@ -13,7 +13,7 @@ function getServiceByGatewayAccountId (gatewayAccountId, correlationId) {
   return baseClient.get({
     baseUrl,
     url: `/services?gatewayAccountId=${gatewayAccountId}`,
-    description: `find a service by it's external id`,
+    description: 'find a service by it\'s external id',
```

Comment:
Possessive ‘its’ shouldn’t even have an apostrophe…

Reply:
It should not but i'm not doing to bother editing that 😆 
Yeh I did, it recognises all the file renames as long as there's no edits which was easier to do because I was using a command line utility to bulk rename